src/azure/CodeGeneratorRba.cs

```diff
@@ -35,6 +46,7 @@ public class CodeGeneratorRba : CodeGeneratorRb
         public override async Task Generate(CodeModel cm)
         {
             var codeModel = cm as CodeModelRba;
+            generatedFolderName = Path.Combine(codeModel.ApiVersion, "generated");

```

Comment:
bit confused here: Is this a path to folder or just the name `generated`

Reply:
I could rename the variable, it's the path to the generated folder, should I call it generatedFolderPath? In terms of the method I'm overriding, I don't think I can rename it, since this is coming from autorest.